### ISSUE TITLE
a11y: Components incorrectly show extreme focus border on mouse click

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#13565]
**Setup**
- Rancher version: v2.11.0 (dev of)

**Describe the bug**
- The same extreme border is shown when the user navigates to an element via keyboard or mouse
- Ideally the extreme border is only shown on keyboard navigation
- Looks like there's a lot of combinations of `&:focus-visible, &:focus` that do this

**To Reproduce**
- Find any input field, click on it

**Result**
![Image](https://github.com/user-attachments/assets/635a28fb-8750-493d-a0c8-07b01031810b)

**Expected Result**
![Image](https://github.com/user-attachments/assets/c8286817-db59-40c7-9e0e-838aa5514eb4)

**Additional context**
`focus-visible` is the one that triggers on keyboard nav and not mouse click

**Comment by aalves08** [comment]
Related to #13522 .

**Comment by gaktive** [comment]
@aalves08 should we have QA validate this as part of #13522 testing then?

**Comment by richard-cox** [comment]
This one is slightly different to #13522, it covers the behaviour of focus rather than just the visuals

**Comment by gaktive** [comment]
Pushing to 2.12.0 as time is running out for 2.11.0.

**Comment by rak-phillip** [comment]
I'm in favor of closing, but I will defer to @edenhernandez-suse for the final judgement call on the updated styles.

**Comment by edenhernandez-suse** [comment]
Sorry, this got lost under other emails. We do want larger borders on focus, simply changing the colour is insufficient for many users. The colour will be updated with the new themes, but the width is fine.

**Comment by richard-cox** [comment]
To re-iterate https://github.com/rancher/dashboard/issues/13565#issuecomment-2707179818.

This is not about changing the size or colour of the accessibility focus border, but when it's shown.

It feels like we should have a normal focus border (indicator?) when initiated by a mouse and the bigger standout focus border when triggered by keyboard navigation. This is a general pattern used on a lot of other sites

**Comment by edenhernandez-suse** [comment]
Both approaches are common practices, but how well they work depends a lot on the colour scheme of the site. For instance, GitHub is using this new approach:

![Image](https://github.com/user-attachments/assets/00e4d687-cc73-457a-801a-11a1080f3313)

![Image](https://github.com/user-attachments/assets/adc9ab5b-ccfa-4ecf-b1e7-7689f88add8f)

You can see with the inspector that they are forcing the focus-visible state on mouse focus. In cluttered UIs, this becomes almost the only way to highlight a field. In lighter UIs with a highly contrasted accent colour, simply changing the colour can work.

**Comment by rak-phillip** [comment]
Let's move this to the UX board if we want to take another design pass on the input focus rings. Otherwise, I think it's safe to close this issue because the implementation adheres to the existing design.

@richard-cox @edenhernandez-suse what do you think?
